### ISSUE TITLE
[Improvement] - Camera frame UI

### ISSUE THREAD

**Issue by feeef** [pencil2d/pencil#451]
This is an improvement I wish to make because I am not fan of the current behaviour.

Instead of having a button in the Display box to toggle camera frame visibility,
![screenshot from 2016-01-21 09-35-17](https://cloud.githubusercontent.com/assets/383399/12475426/3ba70890-c023-11e5-83c9-e65387aa9900.png)

Why not using the existing visibility button on the timeline ?
![screenshot from 2016-01-21 09-35-30](https://cloud.githubusercontent.com/assets/383399/12475456/5b7e73f6-c023-11e5-951e-3388a79c5f2c.png)

This way, if we have 2 different cameras (for 2 different output), we can easily choose to set one visible,
![screenshot from 2016-01-21 09-36-19](https://cloud.githubusercontent.com/assets/383399/12475470/73926ff6-c023-11e5-94a6-71382abdf8d3.png)

Or both of them.
![screenshot from 2016-01-21 09-36-33](https://cloud.githubusercontent.com/assets/383399/12475475/799dfdf2-c023-11e5-89e9-8fad3c42d7f2.png)

**Comment by CandyFace** [comment]
Yes this sounds like a good improvement and frankly.. it makes sense. The icon itself could be a bit nicer too.. i'm thinking an eye like what you're used to.. but in a very simple manner.. I will try to cook something together, that looks sharp and modern.

**Comment by feeef** [comment]
That's great @CandyFace . However, this visibility "dot" has 3 positions (black, grey, white). Maybe an eye wouldn't be ideal.

**Comment by CandyFace** [comment]
Yes i understand that, but really, is that necessary? Two states should be enough to convey visibility, maybe a second icon should be added to let you know which layer is currently focused, or maybe two states are enough. You also have the flat color to gradient transition when switching to a different layer. Maybe that transition could be changed to something less apparent? Just mentioning a few ideas that could help simplifying and make the readability better.

**Comment by feeef** [comment]
Yes, I wish to keep the 3 states because the other layers use the 3 states. We could imagine that in the gray state, the camera view and the dark boarder is pretty transparent. In the black state, the camera border, still gray, is not transparent at all.

**Comment by alexandervrs** [comment]
I am thinking that you could have an eye that is fully open, mid open and closed if you need to have 3 states. I could possibly design an icon for it, not sure of the size/format though.

**Comment by Jose-Moreno** [comment]
I like the "eye" Idea, although I'd suggest an open eye to view layers normally, a "blind" eye to isolate a layer (this function already exists in pencil2d) and "no eye" to hide the layer like this:

![Layer Icon Proposal](http://i.imgur.com/zMoSdZu.jpg)

I think we first have to establish what the "icon placeholder" size will be inside the layer itself, but I think it's safe to create vector artwork in powers of 2 (24x24, 32x32, 64x64, etc) So it can be resized later without issues.

**Comment by Jose-Moreno** [comment]
Also I think it's worth noting that there's a button that allows you to change functionalities between "transparent layer visualization" and "opaque layer visualization" so to speak:

If you make it black all layers will have only two states, visible (on) or invisible (off).

If you make it grey, all layers will have three states, visible / opaque (on), visible / transparent (on) and invisible (off)

I'll record a video of this because it's easier, but I think we should preserve this. The layer isolation that i spoke in my earlier comment is not exactly designed to work like that but to me it's a similar behaviour:

[![Pencil2D Layer States](http://i.imgur.com/qhoMhbz.png)](https://www.youtube.com/watch?v=jJxJ4bGrt3U)

I think we should maintain these states as well. With the icons it should become clearer (but we still need tutorials for Pencil2D hah)
